src/Kafka/KafkaInstanceDrawer/__snapshots__/kafkaConnectionTab.test.tsx.snap

```diff
@@ -86,20 +86,117 @@ exports[`ConnectionTab renders 1`] = `
         class="pf-c-content pf-u-pb-sm"
       >
         <h3
-          class="pf-u-mt-xl"
+          class="pf-u-mt-lg"
```

Comment:
I would change this back to `-xl` to be consistent with the margin-top that's applied to all `h3` elements in this panel.